tests/unit/test_eq.py

```diff
@@ -290,12 +290,10 @@ def test_build_display_address(self):
     def test_build_display_address_raises(self):
         from app import eq
 
-        attributes = self.sample_attributes_json['attributes'].copy()
-        del attributes['Prem2']
+        attributes = {}
 
-        with self.assertRaises(InvalidEqPayLoad) as ex:
+        with self.assertRaises(InvalidEqPayLoad):
             eq.EqPayloadConstructor.build_display_address(attributes)
-        self.assertIn("Prem2 missing from sample attributes", ex.exception.message)
```

Comment:
Could add this back in as:

`self.assertIn("Displayable address not in sample attributes", ex.exception.message)`

Reply:
Done